pytorch_lightning/trainer/trainer.py

```diff
@@ -1734,14 +1734,15 @@ def _log_device_info(self) -> None:
 
         if torch.cuda.is_available() and self._device_type != _AcceleratorType.GPU:
             rank_zero_warn(
-                "GPU available but not used. Set the gpus flag in your trainer `Trainer(gpus=1)` or script `--gpus=1`.",
+                "GPU available but not used. Set `gpus` via your trainer `Trainer(accelerator='gpu', devices=1)`"
+                " or script `--accelerator='gpu' --devices=1`.",
```

Comment:
```suggestion
                "GPU available but not used. Set `accelerator` and `devices` via your trainer
                " `Trainer(accelerator='gpu', devices=1)`.",
```
should we mention about script explicitly? I think it depends upon what kind of argument parser and config they are using...

Reply:
Updated!